tests/functional/pv/space_reclaim/test_disable_reclaimspace_operation.py

```diff
@@ -100,6 +100,7 @@ def execute_reclaimspace_test(self, pod_factory, suspend_state):
             self.wait_till_expected_image_size(pvc_obj, expected_volume_size)
 
     @pytest.mark.polarion_id("OCS-6279")
+    @skipif_external_mode
```

Comment:
Isn't the feature supported in external mode ?

Reply:
@paraggit please check